src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageExtensions.cs

```diff
@@ -74,13 +75,174 @@ public static class LoggerConfigurationAzureTableStorageExtensions
+        {
+            if (loggerConfiguration == null) throw new ArgumentNullException(nameof(loggerConfiguration));
+            if (string.IsNullOrEmpty(connectionString)) throw new ArgumentNullException(nameof(connectionString));
+            
+            try
+            {
+                var storageAccount = CloudStorageAccount.Parse(connectionString);
+                return AzureTableStorage(loggerConfiguration, null, storageAccount, restrictedToMinimumLevel, formatProvider, storageTableName, writeInBatches, period, batchPostingLimit, keyGenerator, bypassTableCreationValidation, cloudTableProvider);
+            }
+            catch (Exception ex)
+            {
+                Debugging.SelfLog.WriteLine($"Error configuring AzureTableStorage: {ex}");
+
+                ILogEventSink sink = new LoggerConfiguration().CreateLogger();
+                return loggerConfiguration.Sink(sink, restrictedToMinimumLevel);
+            }
+        }
+
+        /// <summary>
+        /// Adds a sink that writes log events as records in Azure Table Storage table (default name LogEventEntity) using the given
+        /// storage account name and Shared Access Signature (SAS) URL.
+        /// </summary>
+        /// <param name="loggerConfiguration">The logger configuration.</param>
+        /// <param name="sharedAccessSignature">The storage account/table SAS key.</param>
+        /// <param name="accountName">The storage account name.</param>
+        /// <param name="tableEndpoint">The (optional) table endpoint. Only needed for testing.</param>        /// 
+        /// <param name="restrictedToMinimumLevel">The minimum log event level required in order to write an event to the sink.</param>
+        /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
+        /// <param name="storageTableName">Table name that log entries will be written to. Note: Optional, setting this may impact performance</param>
+        /// <param name="writeInBatches">Use a periodic batching sink, as opposed to a synchronous one-at-a-time sink; this alters the partition
+        /// key used for the events so is not enabled by default.</param>
+        /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
+        /// <param name="period">The time to wait between checking for event batches.</param>
+        /// <param name="keyGenerator">The key generator used to create the PartitionKey and the RowKey for each log entry</param>
+        /// <param name="cloudTableProvider">Cloud table provider to get current log table.</param>
+        /// <returns>Logger configuration, allowing configuration to continue.</returns>
+        /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
+        [Obsolete("Use the newer overload specifying ITextFormatter instead")]
+        public static LoggerConfiguration AzureTableStorage(
```

Comment:
Do these need to be obsoleted? Not against it, but having to understand how to supply an `ITextFormatter` could be a barrier for some people coming to this fresh.